### ISSUE TITLE
Object Interaction

### ISSUE THREAD

**Issue by ghost** [PlaceholderGames/2016-VnD_Game#10]
This issue will be dedicated to how the ants interact with objects in their environment, this can include: wood, stones, food, skill related collectables, possible enemies such as grasshoppers.

**Comment by BojanStankovic** [comment]
This is a bit to broad for an issue. We need to break this into smaller chunks and close this one. Ryan is already working on digging issue which is  encapsulated here as well

**Comment by elliotnaylor** [comment]
Actually reading this issue again, coins might be a really good idea.  So there are more difficult paths that the user can take to get rewards.  It can be added at a much later date

**Comment by ghost** [comment]
I agree its a good idea to include collectables to get the player to choose harder methods of completing the level, in the future its possible to include trophies into the game for collecting all the collectables, this would be a cherry.

**Comment by rbakerusw** [comment]
Should we close this issue? I dont think anyone is working on any object interaction right now and since we are 4 days away from the deadline is would seem pointless to try it.
